docs/DeveloperGuide.md

```diff
@@ -949,6 +1267,162 @@ Although all fields can be edited, the `Name`, `Amount`, and `Type` fields canno
 out in the **Add Transaction** section above also remain.
 </div>
 
+
+#### Clear Transactions
+
+**Overview**
+
+The `clear_transactions` command deletes all existing `Transactions` from `TransactionList` in UniCash.
+
+The activity diagram of clearing all transactions is as shown below
+
+<img src="images/unicash/diagrams/ClearTransactionsActivityDiagram.png" width="450" />
+
+The following sequence diagram shows the interaction between different components of UniCash.
+
+<img src="images/unicash/diagrams/ClearTransactionsSequenceDiagram.png" width="800" />
+
+
+**Details**
+
+1. The user inputs the command to clear all transactions
+2. The input will be parsed by `ClearTransactionsCommandParser` and if the provided input is
+invalid (i.e. trailing arguments) `ParseException` will be thrown, and the 
+user is prompted to enter the command again with the correct input. 
+3. If the input is valid, a `ClearTransactionsCommand` object is created to be executed by `LogicManager`
+4. `LogicManager` will invoke the `execute` method of `ClearTransactionsCommand`
+   which will replace the existing `Model` property with a new `UniCash` object which
+   would contain an empty `TransactionList`.
+
+Here, it must be noted that unlike `DeleteTransactionCommand`, individual transactions in the `TransactionList`
+are not deleted singularly. As opposed to iteratively deleting each transaction in the `TransactionList`, the more
+efficient way to achieve the same effect would be to simply set the `Model` contained in `LogicManager` to a new
+`UniCash` object, as the newly created `UniCash` object would now have an empty `TransactionList` encapsulated within.
+This emulates the deletion of all transactions in the `TransactionList`.
+
+#### Get Transaction
+
+**Overview**
+
+The `get` command retrives an existing `Transaction` from `TransactionList` in UniCash.
+
+The activity diagram of retrieving a Transaction is as shown below
+
+<img src="images/unicash/diagrams/GetTransactionActivityDiagram.png" width="500" />
+
+The following sequence diagram shows the interaction between different components of UniCash.
+
+<img src="images/unicash/diagrams/GetTransactionSequenceDiagram.png" width="1200" />
+
+The above sequence diagram omits details on the filtering of `TransactionList` and assumes that
+the displayed `TransactionList` is showing all transactions. However, the logic of the `DeleteCommand`
+remains the same for all list deletion.
+
+ℹ️ **Note:** The lifeline for `GetCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML,
+the lifeline reaches the end of diagram.
```

Comment:
Might be better to use callouts for this. I've replaced all the ℹ️ **Note:** without callout-info for the edit and summary features

Reply:
Yup, I'm doing that for all my other commands also, thanks